backend/models/job.js

```diff
@@ -0,0 +1,56 @@
+// noinspection SpellCheckingInspection
+
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+
+const JobSchema = new Schema({
```

Comment:
Not sure if we already discussed this, but how are we keeping track of the final driver that is chosen for the jobs. Perhaps we can add a field.

Reply:
This has been added as a non required field in my latest commit